assets/_sass/base/_variables.scss

```diff
@@ -1,14 +1,13 @@
 // 18F Fonts
 $raleway: Raleway,'Helvetica Neue',Helvetica,Arial,sans-serif;
 $open-sans: 'Open Sans','Helvetica Neue',Helvetica,Arial,sans-serif;
-$brand: '18F Nimbus','Helvetica Neue',Helvetica,sans-serif;
-$nimbus: 'Nimbus','Helvetica Neue',Helvetica,sans-serif;
+$brand: 'Nimbus','Helvetica Neue',Helvetica,sans-serif;
```

Comment:
@gboone is this now the original unedited Nimbus? Or the customized 18F Nimbus? It looks like the 18FNimbus files were removed